app/src/main/java/com/dcrandroid/fragments/SendFragment.kt

```diff
@@ -325,6 +325,11 @@ class SendFragment : Fragment(), AdapterView.OnItemSelectedListener {
     }
 
     private fun constructTransaction() {
+        if (activity != null && activity is MainActivity) {
+            val mainActivity = activity as MainActivity
+            if(mainActivity.peerCount < 1) return send_error_label.setText(R.string.not_connected)
```

Comment:
This check should instead be made before prompting the user for password [SendFragment.kt:173](https://github.com/decred/dcrandroid/blob/master/app/src/main/java/com/dcrandroid/fragments/SendFragment.kt#L173). Estimates should be available even if the wallet is not connected.
Also check if the wallet is synced before attempting to send.